ReleaseNotes.md

```diff
@@ -17,55 +15,53 @@ These release notes cover the following topics:
+This package consists of Apache HTTP Server (httpd), along with a number of frequently updated library components (dependencies) required by httpd or one of its optional modules. The package was assembled with VMware's [https://github.com/appsuite/oss-httpd-build](https://github.com/appsuite/oss-httpd-build) OSS Build and Server Instance Schema for Apache HTTP Server.
 
-Unlike many httpd distributions, the end user instance configuration, server content and logs and are not modified in this directory tree. See the section on Instance Creation for details of creating a server instance with these user maintained files.
+This package is structured to allow parallel installation of multiple releases of Apache HTTP Server and related components. It contains one directory tree, labeled as httpd-{version}-{date tag} which represents the version of httpd and the effective date of all components bundled in the package.
 
-In order to build httpd from scratch, see additional details at Pivotal's [https://github.com/appsuite/oss-httpd-build](github oss-httpd-build) project. A tarball of unix sources and zipfile of windows sources is provided alongside the binary release downloads, for ready reference.
+Unlike many httpd distributions, the end user instance configuration, server content and logs and are not modified in this program distribution tree. See the section on Instance Creation for details of creating a server instance with these user maintained files.
 
-Note that Pivotal convenience packages prior to 2.4.37 included OpenSSL 1.1.0, while 2.4.37 and later include OpenSSL 1.1.1. This may cause issues for users who have compiled third-party modules linked to OpenSSL using the apxs utility and openssl library included with an older package. Users are advised to rebuild any such modules before combining them with these newer packages.
+In order to build httpd from scratch, see additional details at VMware's [https://github.com/appsuite/oss-httpd-build](github oss-httpd-build) project. A tarball of unix sources and zipfile of windows sources is provided alongside the binary release downloads, for ready reference.
 
 ## **Included Components**
 
-The following components are included in this 2.4.41-190812 build; those marked (\*) are not compiled on RHEL 7 and Ubuntu 16.04, but the OS Vendor's distribution packages are used, instead. Links to the user change notes and vulnerability indexes are illustrated below.
+The following components are included in this build; those marked (\*) are not compiled for modern linux distributions, but the OS Vendor's distribution packages are used instead. Links to the user change notes and vulnerability indexes are illustrated below.

```

Comment:
Do you need the leading slash? 
"those marked (\*) are"

Reply:
and the ", but" clause reads awkwardly

Reply:
It's md syntax. There's no * shown when examining this at https://github.com/appsuite/oss-httpd-build/blob/master/doc/Release%20Notes%20for%20Pivotal%20Build%20of%20Apache%20HTTP%20Server.md#included-components

(I've considered changing this to an .adoc, but not today.)